src/components/quotes/Quotes.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import './quotes.css';
+
+const apiUrl = process.env.REACT_APP_API_URL;
+const apiKey = process.env.REACT_APP_API_KEY;
```

Comment:
:orange_circle: [Optional] Currently, the API key and URL are stored in environment variables (process.env.REACT_APP_API_URL and process.env.REACT_APP_API_KEY), which is a good practice for security. However, to make it easier for others to test your project without exposing your API key, you can provide a default configuration or instructions for setting up their own environment variable
Default Configuration: In your project's code, you can set default values for the API URL and key. These defaults can be used if the environment variables are not set. For example:
```

const apiUrl = process.env.REACT_APP_API_URL || 'https://example.com/api';
const apiKey = process.env.REACT_APP_API_KEY || 'your-api-key';
```


Your project's reliance on environment variables without clear instructions or default configurations can be a significant reason why it might not work on someone else's machine. To improve the project's usability, consider providing default values for the API URL and key in your code, and offer clear instructions in the documentation or README on how others can set up their own environment variables for testing purposes. This will make it easier for others to work on your project without exposing your API key. 

NOTE:  **I appreciate your effort in providing a video demonstration of the project's functionality, which helped me understand how it works on your computer. However, it's crucial to ensure that the project is accessible and usable by others without the need for extensive setup or additional resources.**

Reply:
Alright, thank you for highlighting that